planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2494,8 +2573,15 @@ void AvoidanceModule::CompensateDetectionLost(ObjectDataArray & now_objects) con
       n.begin(), n.end(), [&r_id](const auto & o) { return o.object.object_id == r_id; });
   };
 
+  const auto isIgnoreObject = [&](const auto & r_id) {
```

Comment:
This actually searches the already registered object right?
I suggest `isAlreadyRegistered`, but in case it is not suitable, then just ignore this.

Reply:
Originally, this fuction compensates not only detection lost but also objects that not satisfy condition for avoidance target (e.g. object starts moving). I have no comment to the former, but I don't think that the module don't have to compensate those objects in the latter case. What would you think?

Reply:
I see, I understand the reason for it now.